### PR TITLE
README: fixes some grammatical errors

### PR DESCRIPTION
**Pull request by arshPratap**
Fixed some grammatical errors found in the readme file

### CODE REVIEW

**Review by khancyr — APPROVED** [review]
I am not english, but that seems good

**Comment by rmackay9** [comment]
Looks good, can you change the commit line to be "README: fixes some grammatical errors"?  to be clear I mean fix the commit description itself not the PR's description.

**Comment by peterbarker** [comment]
`README.md: fix some grammatical errors`

**Comment by arshPratap** [comment]
Thank you @khancyr, @peterbarker, @rmackay9 for your tips and suggestions. I have made the necessary changes and am looking forward to contribute to Ardupilot.

**Comment by khancyr** [comment]
Hello,

Thank you. You are close to get it right. We don't use the merge commit, so you will need to do a rebase on our master branch to get rid of it and keep only your commit !

You can look on our wiki on how to do that, in the dev section

**Comment by arshPratap** [comment]
Sir, I have done the necessary changes

**Comment by arshPratap** [comment]
Please take a look..Thanks a lot in advance

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
 been under development since 2010 by a team of diverse professional engineers and computer scientists. It is
 the only autopilot software capable of controlling almost any vehicle system imaginable, from conventional
-airplanes, multirotors, and helicopters, to boats and even submarines. And now being expanded to feature
+airplanes, multirotors, and helicopters, to boats and even submarines. And is now being expanded to provide
```
Comment (eric-wieser) [inline README.md:14]:
```suggestion
airplanes, multirotors, and helicopters, to boats and even submarines. It is now being expanded to provide
```

Comment (Hwurzburg) [inline README.md:14]: if we are going to update, might as well make it up to date on vehicle support also...
```suggestion
airplanes, quadplanes, multirotors, and helicopters, to rovers, boats, balancebots, and even submarines. And is now being expanded to provide
```

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
 support for new emerging vehicle types such as quad-planes and compound helicopters.
```
Comment (Hwurzburg) [inline README.md:15]:
```suggestion
support for new emerging vehicle types such as compound helicopters.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by magicrub** [comment]
This is 3 commits for 2 lines (one chunk) of text. Needs to be squashed.

**Comment by arshPratap** [comment]
OK sir, I will squash the commits

**Comment by arshPratap** [comment]
Sir, I have squashed the above commits into one commit and I want to thank @eric-wieser and @Hwurzburg for suggesting the changes.

**Comment by magicrub** [comment]
Merged, thank you!

**Pull request merged**